lib/db/feature-tag-store.js

```diff
@@ -62,131 +28,36 @@ class FeatureTagStore {
         return rows.map(this.featureTagRowToTag);
     }
 
-    async getTagByTypeAndValue(type, value) {
-        const stopTimer = this.timer('getTagByTypeAndValue');
-        const row = await this.db
-            .first(COLUMNS)
-            .from(TABLE)
-            .where({ type, value });
-        stopTimer();
-        if (!row) {
-            throw new NotFoundError('Could not find this tag');
-        }
-        return this.rowToTag(row);
-    }
-
-    async hasTag(tag) {
-        const stopTimer = this.timer('hasTag');
-        const row = await this.db
-            .first(COLUMNS)
-            .from(TABLE)
-            .where({
-                type: tag.type,
-                value: tag.value,
-            });
-        stopTimer();
-        if (!row) {
-            throw new NotFoundError('No tag found');
-        }
-        return this.rowToTag(row);
-    }
-
-    async tagFeature(event) {
+    async tagFeature(featureName, tag) {
         const stopTimer = this.timer('tagFeature');
-        const tag = this.eventDataToRow(event);
-        try {
-            await this.hasTag(tag);
-        } catch (err) {
-            if (err instanceof NotFoundError) {
-                this.logger.info(`Tag ${tag} did not exist. Creating.`);
-                await this._createTag(tag);
-            } else {
-                this.logger.debug('Already existed');
-            }
-        }
         await this.db(FEATURE_TAG_TABLE)
             .insert({
-                feature_name: event.featureName,
+                feature_name: featureName,
                 tag_type: tag.type,
                 tag_value: tag.value,
             })
```

Comment:
would it make sense to have a small function converting from featureName and tag to column names? You do the same at line 45

Reply:
Yes, it probably would, I'll add it